util/misc.py

```diff
@@ -8,8 +8,83 @@
 import time
 from collections import OrderedDict
 
+import constants
 
-def last_run_time_acceptable(coll, found, last_run, config_backoff_time):
+
+def check_data_package_system_avus(extracted_avus):
```

Comment:
Should we have another separate util avu file for just non-irods dependent functions?